infrastructure/provena/config/config_helpers.py

```diff
@@ -164,34 +164,42 @@ def subdomain_to_full(subdomain: str) -> str:
     return ResolvedDomainNames(
         # root
         root_domain=config.dns.root_domain,
-
         # apis
-        data_store_api=subdomain_to_full(
-            config.components.data_store.api_domain) if config.components.data_store else "",
-        prov_api=subdomain_to_full(
-            config.components.prov_store.api_domain) if config.components.prov_store else "",
-        registry_api=subdomain_to_full(
-            config.components.entity_registry.api_domain) if config.components.entity_registry else "",
-        search_api=subdomain_to_full(
-            config.components.search.api_domain) if config.components.search else "",
-        auth_api=subdomain_to_full(
-            config.components.auth_api.api_domain) if config.components.auth_api else "",
-        warmer_api=subdomain_to_full(
-            config.components.warmer.domain) if config.components.warmer else "",
-        async_jobs_api=subdomain_to_full(
-            config.components.async_jobs.job_api_domain) if config.components.async_jobs else "",
-
+        data_store_api=subdomain_to_full(config.components.data_store.api_domain)
+        if config.components.data_store
+        else "",
+        prov_api=subdomain_to_full(config.components.prov_store.api_domain)
+        if config.components.prov_store
+        else "",
+        registry_api=subdomain_to_full(config.components.entity_registry.api_domain)
+        if config.components.entity_registry
+        else "",
+        search_api=subdomain_to_full(config.components.search.api_domain)
+        if config.components.search
+        else "",
+        auth_api=subdomain_to_full(config.components.auth_api.api_domain)
+        if config.components.auth_api
+        else "",
+        warmer_api=subdomain_to_full(config.components.warmer.domain)
+        if config.components.warmer
+        else "",
+        async_jobs_api=subdomain_to_full(config.components.async_jobs.job_api_domain)
+        if config.components.async_jobs
+        else "",
```

Comment:
This is a bit weird, but i guess the attribute for domain is diffferent for different components so cant really be have a generic function for all these if else statements.